posts/2024-01-31-Performance-Doris_Amoakohene/index.qmd

```diff
@@ -0,0 +1,247 @@
+---
+title: "Visualizing Performance Regression of data.table with Atime"
+author: "Doris Afriyie Amoakohene"
+date: "2024-02-04"
+output: html_document
+---
+
+
+Since August 2023, I have been working on performance testing, which could be useful for expanding the open-source Ecosystem around data.table package in R,This could increase confidence in code contributions by ensuring the sustained efficiency of the data.table package. Consequently, it becomes essential to enhance the package's performance and deliver reliable results to its users.
```

Comment:
> open-source Ecosystem around data.table package in R,This

open-source ecosystem around data.table package in R. This

Reply:
done